ccd-definition/FixedLists.json

```diff
@@ -430,5 +430,11 @@
     "ID": "RelevantProceedings",
     "ListElementCode": "Yes",
     "ListElement": "Yes"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ApplicationDeletion",
+    "ListElementCode": "Delete",
```

Comment:
Value does not matter here. Can we make it empty?

Reply:
unsure what you mean here, remove ListElementCode?

Reply:
I meant having empty code like:

```"ListElementCode": ""```

Reply:
**"ListElementCode": "" gives:**
import fails with Couldn't find the column ListElementCode in the sheet FixedLists

**"ListElementCode": " " gives:**
There's a missing value in the column 'ListElementCode' or invalid value in the sheet 'FixedLists'

Reply:
I shall leave as is